docs/system_overview/configuration.md

```diff
@@ -913,11 +913,15 @@ The maximum TTL of a randomly generated password in seconds. If a user attempts
 
 The minimum TTL of a randomly generated password in seconds. If a user attempts to authenticate with a TTL value that is less than `password_min_time`, the TTL is determined to be invalid and an error is returned. If `password_min_time` is configured to a value greater than `password_max_time` no passed-in TTL value will satisfy the system. Accepted values: [0..18446744073709552000). If a value outside of the acceptable range is used, a warning message will be logged for the administrator and the default value will be used. The default value is 121.
 
+!!! Note
+    If users are being forced to re-authenticate via PAM frequently, you may need to adjust this option. See [Users are forced to re-authenticate after a few minutes](../../system_overview/troubleshooting/#users-are-forced-to-re-authenticate-after-a-few-minutes) for more information.
```

Comment:
Maybe mention `CAT_PASSWORD_EXPIRED` here somewhere so that it can be found in a search. Might be worth including that error message explicitly in the Troubleshooting section as well.

Reply:
done